tests/kernel/kirk.pm

```diff
@@ -13,12 +13,16 @@ use strict;
 use testapi;
 use warnings;
 use serial_terminal 'select_serial_terminal';
+use version_utils qw(is_transactional);
+use transactional 'trup_install';
+use LTP::utils;
```

Comment:
Wouldn't be better to create function for kirk installation and move it to utils as well?

Reply:
Same concept of `install_ltp.pm` ?